scripts/test-mps-valid-avro.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+"""A one off script to test whether mozilla-pipeline-schemas are generated
+as valid avro schemas. This script requires avro to be installed, as well
+as running the `download-mps.sh` script."""
+
+import json
+import logging
+import os
+import subprocess
+import avro.schema
+
+
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+BIN = "target/debug/jsonschema_transpiler"
+
+
+def build():
+    """Initialize a build of the transpiler."""
+    subprocess.call(["cargo", "build"])
+
+
+def avro_schema(path):
+    """Return an avro schema in json if valid, None otherwise."""
+    schema = None
+    try:
+        data = subprocess.check_output(
+            [BIN, "--from-file", path, "--type", "avro"], stderr=subprocess.DEVNULL
+        )
+        schema = json.loads(data)
+    except subprocess.CalledProcessError:
+        pass
+    return schema
+
+
+def parse_schema_name(path):
+    """Given a directory path to a json schema in the mps directory, generate
+    the fully qualified name in the form `{namespace}.{doctype}.{docver}`."""
+    elements = path.split("/")
+    doctype, docver = elements[-1].split(".")[:-2]
+    namespace = elements[-3]
+    return f"{namespace}.{doctype}.{docver}"
```

Comment:
Python noob here, but curious: What does the `f` prefix do?

Reply:
It's the python 3 syntax for literal string interpolation ([pep 498](https://www.python.org/dev/peps/pep-0498/)). It's clean.